pages/landing/hipaa/template/index.html

```diff
@@ -0,0 +1,66 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Achieve HIPAA compliant infrastructure, instantly
+excerpt: Go live with a customized, HIPAA-compliant cloud infrastructure and sample application in about a day.
+permalink: /hipaa-cloud-compliance/
+
+hero:
+  image: /assets/img/infrastructure-cube-icons/infrastructure-in-a-day.png
+  cta: Join the Waitlist
+  logos:
+    - /assets/img/logos-technologies/partner_logo_aws.png
+    - /assets/img/logos-compliance/cis.png
+    - /assets/img/logos-compliance/hipaa.png
+
+how_it_works:
+  - heading: Deploy on HIPAA-compliant infrastructure
+    image: /assets/img/infrastructure-cube-icons/infrastructure-hipaa.png
+    image_width: 379px
+    image_side: right
+    body: |
+      We are experts in Cloud infrastructure and AWS. Our IaC library and architectures are certified HIPAA-complaint via third party audit.
+
+  - heading: Build on strong foundations with our HIPAA-compliant sample app
+    image: /assets/img/infrastructure-cube-icons/hipaa-sample-app.png
+    image_side: left
+    body: |
+      Start off on the right foot. We've already built world-class, HIPAA-compliant sample apps certified via third party audit — just add your business logic.
```

Comment:
1. :+1: 
2. Hmm - "starter app?" I like scaffold because it's accurate, but I wonder if it's less legible for non-developers? App harness, compliant app skeleton, compliant application starter...
3. Good idea - will add to the product details section
4. Initially spelled this out in the product details section.

Reply:
Added the info for 2,3,4 to the product details section in f5869e7f.

Reply:
Starter app still puts the focus on "starting" rather than "production-grade." Sadly, all the "5 minute demo" focus of most of the software industry has led to most of these terms implying "good to start with, but not really meant for prod." There must be some terminology that communicates what we want, I'm just struggling to come up with it. 

Best ideas so far:

1. Scaffolding
1. Skeleton
1. Production-hardened application skeleton
1. Application template
1. Production application template
1. Hardened application template
1. Usage pattern
1. Design pattern

Reply:
I think Scaffolding is still accurate but I wonder how universally parseable it is. HIPAA-compliant application template, however, to me means a working application that I can modify to my own purposes. 

Pattern feels less valuable to me since a given app can implement many design patterns - some of which might ultimately be antipatterns. So on the whole I perceive it as less exciting and less complete than a "template".

Reply:
For the time being we decided on "HIPAA-compliant application template" - but can switch it out if there's a preferred phrase.